server/database/queries/users/addUserInterests.js

```diff
@@ -0,0 +1,14 @@
+const dbConnection = require('../../config/dbConnection');
+
+const interestsQueryMaker = (interests) => {
+  const query = 'INSERT INTO user_interest(interest_id, user_id) VALUES ($1,$2)';
```

Comment:
```suggestion
 const query = 'INSERT INTO user_interest( user_id,interest_id) VALUES ($1,$2)';
```

**the `user_id` should be the first inserted** 
if you look to the line 5 you will find that we provided as the first parameter to the values